.github/workflows/release.yaml

```diff
@@ -85,12 +85,21 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Build and push image to Docker Hub
+      # <--- build and push kusion image to Dockerhub --->
+      - name: Build and push kusion image to Docker Hub
         uses: docker/build-push-action@v3
         with:
           context: .
           push: true
           tags: kusionstack/kusion:latest,kusionstack/kusion:${{ steps.get_version.outputs.kusion_version }}
+      # <--- build and push kusionctl image to Dockerhub --->
+      - name: Build and push kusionctl image to Docker Hub
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: Dockerfile_kusionctl
+          push: true
+          tags: kusionstack/kusionctl:latest,kusionstack/kusionctl:${{ steps.get_version.outputs.kusion_version }}
```

Comment:
how about rename to kusion-mini or kusion-lite ?

Reply:
kusion-lite seems good to me, how do u think @elliotxx

Reply:
i think kusionctl is good, easy to understand

Reply:
Thanks for all the good opinions. since this image is barely about kusionctl binary, I think kusionctl is more precisely here. and the name "kusion-lite" can be preserved to represent the light version of the kusion pack (might have more than one kusionctl binary, but with some other tools).